src/megameklab/com/printing/PrintEntity.java

```diff
@@ -152,6 +153,23 @@ public String formatQuirks() {
     public String formatTacticalFuel() {
         return "";
     }
+
+    /**
+     * Converts a weight to a String, either in kg or tons as appropriate to the Entity,
+     * labeled with the measurement unit.
+     * @param weight The weight in tons
+     * @return       The formatted weight with units
+     */
+    String formatWeight(double weight) {
```

Comment:
Just checking to make sure I understand this correctly, this means that is package access? Given both PrintTank and PrintAero extend PrintEntity, why not make this protected?